### ISSUE TITLE
Audio Features Deprecated? 403 Response

### ISSUE THREAD

**Issue by s2t2** [spotipy-dev/spotipy#1173]
I am trying to get audio features for a given track, using the `audio_features` method.

```
creds = SpotifyClientCredentials(client_id=client_id, client_secret=client_secret)
client = Spotify(client_credentials_manager=creds)

track_ids = ["5xTtaWoae3wi06K5WfVUUH"]

client.audio_features(track_ids) #> 403
```

However it is returning a 403.

I was able to fetch normal metadata about the track using the `track` method without any issues.

I thought maybe using my user access token auth might help, but no luck.

I read this method may have been deprecated?
https://developer.spotify.com/documentation/web-api/reference/get-audio-features

Any assistance or information would be appreciated. Thank you!

**Comment by AlexanderWangY** [comment]
https://developer.spotify.com/blog/2024-11-27-changes-to-the-web-api

I believe the changes introduced to the spotify web API have deprecated some of the features you are trying to access. Specifically the audio features method.

**Comment by s2t2** [comment]
😭

**Comment by syphernl** [comment]
See https://github.com/spotipy-dev/spotipy/issues/1172

**Comment by s2t2** [comment]
Really surprising mismanagement of their API without warning or recourse to keep the functionality.

**Comment by lemesdaniel** [comment]
Is there any alternative to get this data somewhere else?

**Comment by dieser-niko** [comment]
> Really surprising mismanagement of their API without warning or recourse to keep the functionality.

Honestly, it's not that surprising anymore. Spotify has done something like this before, changing an endpoint without warning (I think it was deleting a specific track from a playlist).

**Comment by omasseau** [comment]
> for anybody who is interested: https://essentia.upf.edu/models.html
> 
> you can use Essentia which is a library focused on audio analysis. There are a plethora on models u can use to extract features.
> 
> Spotify won't take us down.

But you need the audio file, right ?

**Comment by timtensor** [comment]
> won't

Yes I think so. I was using end points to make customized playlist based on a reference song . Looking for most similar songs but now I don't know how to address that .

**Comment by pkpk-pixel** [comment]
@s2t2 @lemesdaniel @omasseau if you guys are still looking for an alternative checkout https://rapidapi.com/soundnet-soundnet-default/api/track-analysis. Kept my project alive
